### DISCUSSION TITLE
Add v1 "enable wifi" into v2 script

### DISCUSSION THREAD

**Discussion by annoyinganime** [ExpressLRS/ExpressLRS#1492] [Ideas]
I've seen lots of new users buying new tx module, downloading latest script, and wondering why nothing works. What I suggest is adding one backwards compatible option "enable wifi" for modules with v1 firmware. So when someone connects new TX with v1 firmware, and try to use new v2 script - users will see only option "enable wifi", and won't need to separately download v1 script just to enable update

**Comment by schugabe** [comment]
The communication between LUA and TX module was changed completely between v1 and v2. So it would require quite some code on the lua side to add that option back in. The radios don't have unlimited memory and I don't think that this would be a good usage of the resources for this feature: users would still have to update the v2 script to get the new feature and with the same effort they could install the v1 script.
